### PR TITLE
Create drivetrain

### PR DESCRIPTION
**Pull request by FRCTeam3255-Shared-K1-10**

### CODE REVIEW

**Comment by TaylerUva** [comment]
Good except one bug:
https://github.com/FRCTeam3255/2024_Sportbot_Code/actions/runs/11533884009/job/32107301309?pr=7#step:6:39

File: src/main/java/frc/robot/commands/Drive.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    globalDrivetrain.setDrivetrainSpeed(globalForwardSpeed.getAsDouble() - globalRotationSpeed.getAsDouble(), globalForwardSpeed.getAsDouble() + globalRotationSpeed.getAsDouble());
```
Comment (TaylerUva) [inline src/main/java/frc/robot/commands/Drive.java]: I would move the math into the method

**Review by TaylerUva — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Drivetrain.java [hunk 2]
```diff
@@ -23,8 +23,26 @@ public Drivetrain() {
...
   }
 
+  /**
+   * Sets the velocity of the drivetrain motors.
+   * 
+   * @param rightVelocity The velocity to set for the right side of the drivetrain.
+   * @param leftVelocity The velocity to set for the left side of the drivetrain.
+   * @param rotationSpeed The rotation speed to apply to the drivetrain.
+   */
+  public void setDrivetrainSpeed(double rightVelocity, double leftVelocity, double rotationSpeed) {
```
Comment (TaylerUva) [inline src/main/java/frc/robot/subsystems/Drivetrain.java]: Why not just forward velocity and rotate velocity?

Comment (S0L0GUY) [reply src/main/java/frc/robot/subsystems/Drivetrain.java]: That makes sence

**Review by TaylerUva — APPROVED** [review]
Excellent work!

**Comment by TaylerUva** [comment]
@S0L0GUY looks like there are merge conflicts

**Review by TaylerUva — APPROVED** [review]

**Pull request merged**